### ISSUE TITLE
Alarm issue with DST

### ISSUE THREAD

**Issue by aaron-grewell** [philliphsu/ClockPlus#15]
When setting an alarm during DST the hour shown is inconsistent.  The time picker shows a different time than the alarm overview list.

**Comment by philliphsu** [comment]
You're right. Setting times between 12 AM - 2 AM (00:00 - 02:00) ends up creating the alarm with a +1 hour difference. Are you noticing any other affected times?

**Comment by aaron-grewell** [comment]
I set a time of 0700 and got 0800 instead.  This is in PST/PDT, not sure if that matters.

**Comment by philliphsu** [comment]
I can't seem to recreate the issue. What device/OS are you using? Any other times that are wrong?

**Comment by aaron-grewell** [comment]
I'm using CyanogenMod 13 on an LG G3 D850. The Android version is 6.0.1.  I'm not sure about other times, that was the first one I tried to set since DST started.  I'll try some others.

**Comment by philliphsu** [comment]
Device/OS probably isn't a factor. I have a hunch that the issues with setting times between 00:00 - 02:00 might "disappear" after midnight tonight (start of the next day).

**Comment by philliphsu** [comment]
I can confirm the issues with times 00:00 - 02:00 has "disappeared" on its own as of March 13. Please let me know if you are still experiencing problems. I will still attempt to patch this bug.

**Comment by aaron-grewell** [comment]
The issue with my 7am alarm has also disappeared.  One additional note: I created that alarm prior to the start of DST.  I'm not sure if that matters.  Thanks for taking a look at this.

**Comment by revuwa** [comment]
I had exactly the same issue for several days, (before I've stopped using the app).
I've oversleep one day, because I set the alarm to 06:XX and the alarm came up 07:XX. I opened the time settings and saw 07:XX, then I wanted to modify this entry and the time picker sayd it's 06:XX. So I confirmed with OK, and the time of the alrm was 07:XX again * **tilt** *
I made another alarm. Here it worked to set an alarm to 06:XX. After it ringed (to the right time), the settings of the exactly same alarm transformed into 07:XX again. 
That was the point, I uninstalled the app...

If anybody find/fix this issue I really want to give this great app a second chance.
